app/view_models/prior_authority/v1/decision.rb

```diff
@@ -7,7 +7,7 @@ class Decision < BaseViewModel
       delegate :state, to: :submission
 
       def comments
-        submission.latest_decision_event.details['comment']
+        submission.latest_decision_event&.details&.fetch('comment', '')
```

Comment:
is this comment not now stored in the JSON blob?

Reply:
there are multiple values in the JSON blob, so could either rely on latest decision event, as above, or combination of current state plus the relevant attribute below logic. currently latest decision event seems the most sensible to me especially as I am not sure whether a further information request is left in a status of submitted for which there is no  "pending..explanation" state.

```
   # payload change when part granting...
   "pending_granted_explanation"=>"",
   "pending_rejected_explanation"=>"",
   "pending_part_grant_explanation"=>"I have part granted this because....",
```

As an aside, should we event be sending an email immediately, if the decision is "pending"?!

Reply:
Sorry, this is my fault - the reason these are all prefixed with `pending` is that when I wrote this logic, I was thinking of the 'save and come back later' route - we were saving these comments in the blob purely so that we could re-access them when we came back later, so _in that scenario_ these would be pending values because a decision would not have been made. However, those values also get written when we actually finalise the decision, so they are technically available to us here when we're viewing the post-decision screen. But possibly they _shouldn't_ be there - maybe pending decisions should be cleared out when a decision is actually made, in which case the canonical source of truth for the comment is the latest decision event.